### PR TITLE
Split out the Term Graph Loaders #2806

### PR DESCRIPTION
**Pull request by prb112**
- refactor the pom module
- split loader and graph

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: term/fhir-term-loader/pom.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.ibm.fhir</groupId>
+        <artifactId>fhir-term-parent</artifactId>
+        <version>4.10.0-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <artifactId>fhir-term-loader</artifactId>
```
Comment (lmsurpre) [inline term/fhir-term-loader/pom.xml:11]: do we expect this project to evolve beyond loading just the fhir-term-graph?
if not, we might want to use `fhir-term-graph-loader` instead.  I'm good either way thought.

**Review by lmsurpre — APPROVED** [review]
Very glad to have the loader separated from the termProvider.  I'm not sure we needed to move all term-related projects into a subdirectory, but I'm fine with it.  LGTM.

File: term/fhir-term-loader/pom.xml (hunk 1 above)
Comment (prb112) [reply term/fhir-term-loader/pom.xml:11]: Changed to be more specific

**Comment by lmsurpre** [comment]
I missed it / forgot about it in my first review, but we should also update the README to reflect this new module (and the updated paths to the term modules)

**Pull request merged**